src/molecules/FrontSide/index.jsx

```diff
@@ -5,29 +5,65 @@ import * as React from "react";
 import useStore from "../../ions/store/store";
 import { StyledCardMainArea } from "./styled";
 
+import { Link } from "@mui/material";
+//import Speech from "react-speech";
+
 const FrontSide = () => {
 	const currentCard = useStore(state => state.currentCard);
 	const showKana = useStore(store => store.showKana);
 	const showTranslation = useStore(store => store.showTranslation);
 	const learnMode = useStore(state => state.learnMode);
 
+	const speakIt = textToSpeak => {
+		let voices = speechSynthesis.getVoices();
+		let utterance = new SpeechSynthesisUtterance(textToSpeak);
+		utterance.voice = voices[58]; // 18 58
+		utterance.lang = "ja-JP";
+		speechSynthesis.speak(utterance);
+	};
+
 	return (
 		<StyledCardMainArea>
 			<Stack justifyContent="center" alignItems="center" spacing={2}>
 				<CardContent>
 					<Typography gutterBottom align="center" variant="h2" component="div">
-						{currentCard.kanji}
+						<Link
+							color="#616161"
+							underline="none"
+							unselectable="on"
+							onClick={() => {
+								speakIt(currentCard.kanji);
+							}}
+						>
+							{currentCard.kanji}
+						</Link>
```

Comment:
this feature is absolutely awesome, but why dont you use a proper html element to handle the onClick? Something a user can recognize as usable, like a button with a speaker symbol in it. You could easily nest a Button in the next.js Link Component 

`<Link onClick={speak!}><Button>play sound</Button></Link>`